src/cpp/src/llm_pipeline.cpp

```diff
@@ -699,7 +699,8 @@ ov::genai::LLMPipeline::LLMPipeline(
     const ov::AnyMap& properties
 ){
     auto start_time = std::chrono::steady_clock::now();
-    if (properties.find(ov::genai::scheduler_config.name()) != properties.end()) {
+    if (properties.find(ov::genai::scheduler_config.name()) != properties.end() || 
+        properties.find(utils::DRAFT_MODEL_ARG_NAME) != properties.end()) {
```

Comment:
Could you please extend it by `prompt_lookup`?

Reply:
Yes, done.